tools/mkmapdata/mkmapdata.sh

```diff
@@ -288,7 +278,7 @@ else
 fi
 
 # Set timestamp for names below 
```

Comment:
I modified the next line to avoid recreating map files that were created yesterday.

To later return to working with today's date again (as the original code did), I will replace the "-1" with a "0" (in a future commit).

Reply:
Hmmm OK. It actually won't really help, because it's also going to name the output file with the previous days date. So it's just going to have a clock that's 24 hours behind. You'd have to do something more sophisticated to get both today's files and yesterdays files. Having that would be useful if it takes a long time to make maps and you are creating them a lot (like while you are developing this).

Reply:
You're right that this labels new files with yesterday's date, but that was fine because my **only** goal at the time was to avoid duplicating already created files.